app/templates/components/code-theme-selector.hbs

```diff
@@ -0,0 +1 @@
+{{yield}}
```

Comment:
Do we actually have a reason to yield here? As in, do we expect to use the component in a way that provides it HTML to render? If not, we could just have an empty file here instead (and remove the test that ensures yielding works, if there is one).